internal/table.go

```diff
@@ -0,0 +1,192 @@
+	case tea.KeyMsg:
+		switch msg.String() {
+		case "esc":
+			if m.table.Focused() {
+				m.table.Blur()
+			} else {
+				m.table.Focus()
+			}
+		case "q", "ctrl+c":
+			return m, tea.Quit
+			//		case "enter":
+			//			return m, tea.Printf("Selected asset: %s", m.table.SelectedRow()[0])
+		}
+	}
+	m.table, cmd = m.table.Update(msg)
+	return m, cmd
+}
+
+func (m model) View() string {
+	return baseStyle.Render(m.table.View()) + "\n"
+}
+
+func RenderAssetsTable(columns []table.Column, rows []table.Row, height int) {
+	t := table.New(
+		table.WithColumns(columns),
+		table.WithRows(rows),
+		table.WithFocused(true),
+		table.WithHeight(height),
+	)
+
+	// Styling
+	s := table.DefaultStyles()
+	s.Header = s.Header.
+		BorderStyle(lipgloss.NormalBorder()).
+		BorderForeground(lipgloss.Color("#6e6f6eff")).
+		BorderBottom(true).
+		Bold(true)
+	s.Selected = s.Selected.
+		BorderForeground(lipgloss.Color("#6e6f6eff")).
+		Foreground(lipgloss.Color("#ffffff")).
+		Background(lipgloss.Color("#49306d")).
+		Bold(false)
+	t.SetStyles(s)
+
+	m := model{t}
+	if _, err := tea.NewProgram(m).Run(); err != nil {
+		fmt.Println("Error running program:", err)
+		os.Exit(1)
+	}
+}
+
+func RenderAssetsTableOnce(columns []table.Column, rows []table.Row, height int) {
+	t := table.New(
+		table.WithColumns(columns),
+		table.WithRows(rows),
+		table.WithHeight(height),
+	)
+
+	s := table.DefaultStyles()
+	s.Header = s.Header.
+		BorderStyle(lipgloss.NormalBorder()).
+		BorderForeground(lipgloss.Color("#6e6f6eff")).
+		BorderBottom(true).
+		Bold(true)
+	s.Selected = s.Selected.
+		BorderForeground(lipgloss.Color("#6e6f6eff")).
+		Foreground(lipgloss.Color("#ffffff")).
+		Background(lipgloss.Color("#49306d")).
+		Bold(false)
+	t.SetStyles(s)
+
+	fmt.Println(baseStyle.Render(t.View()))
+}
+
+func ShowPassiveScanResults() {
+	columns := []table.Column{
+		{Title: "Src IP", Width: 15},
+		{Title: "Protocol", Width: 10},
+		{Title: "Src MAC", Width: 20},
+		{Title: "Dst MAC", Width: 20},
+		{Title: "EthType", Width: 12},
+	}
+
+	var rows []table.Row
+	for _, r := range passive_results {
+		rows = append(rows, table.Row{
+			r.SrcIP,
+			r.Protocol,
+			r.SrcMAC,
+			r.DstMAC,
+			r.EthernetType,
+		})
+	}
+	RenderAssetsTable(columns, rows, 10)
+}
+
+func ShowAzureResultsTable() {
+	columns := []table.Column{
+		{Title: "Field", Width: 20},
+		{Title: "Value", Width: 100},
+	}
+
+	var rows []table.Row
+	for _, r := range azure_results {
+		rows = append(rows, table.Row{"Name         │", r.Name})
+		rows = append(rows, table.Row{"UniqueID     │", r.UniqueID})
+		rows = append(rows, table.Row{"Location     │", r.Location})
+		rows = append(rows, table.Row{"ResourceGroup│", r.ResourceGroup})
+		rows = append(rows, table.Row{"NIC          │", r.NIC})
+		rows = append(rows, table.Row{"MAC          │", r.MAC})
+		rows = append(rows, table.Row{"Subnet       │", r.Subnet})
+		rows = append(rows, table.Row{"Vnet         │", r.Vnet})
+		rows = append(rows, table.Row{
```

Comment:
Do the same change as mentioned [here](https://github.com/Naman1997/discovr/pull/47/files#r2397000304)